### ISSUE TITLE
Update accessability docs

### ISSUE THREAD

**Issue by razzeee** [recharts/recharts#6058]
The accessibility docs in the storybook https://recharts.org/en-US/storybook seem to be outdated, since the release of 3.0

`accessibilityLayer` should probably be removed? And some of the texts should be revised to reflect the new state.

**Comment by ckifer** [comment]
Thanks - labeled

**Comment by shreedharbhat98** [comment]
@ckifer I would like to work on this. Could you please assign it to me?
Thanks
